docs/hooks/useAlert.md

```diff
@@ -1,13 +1,26 @@
-# `useAlert`
+# useAlert Hook
 
-`useAlert(message, options?) → { show }`
+## Basic Usage:
 
-## Hook arguments
+```js
+import { useAlert } from '@dhis2/app-runtime'
+
+// Within a functional component body
+const { show } = useAlert(message, options)
+```
+
+## Input
 
-| Name      | Type                   | Description                                                                                                                                                                                                                                                              |
-| --------- | ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `message` | `string` or `Function` | The message to display                                                                                                                                                                                                                                                   |
-| `options` | `object` or `Function` | A configuration object that matches [the props of the `AlertBar`](https://ui.dhis2.nu/demo/?path=/docs/feedback-alerts-alert-bar--default) in `@dhis2/ui` (alternate: [minimal view](https://ui.dhis2.nu/#/api?id=coresrcalertbaralertbarproptypes-object) of the props) |
+| Name    | Type                   | Required     | Description                                                                                                             |
+| ------- | ---------------------- | ------------ | ----------------------------------------------------------------------------------------------------------------------- |
+| message | _string_ or _function_ | **required** | The message to display                                                                                                  |
+| options | _object_ or _function_ |              | A configuration object that matches [the props of the `AlertBar`](https://ui.dhis2.nu/#/api?id=alertbar) in `@dhis2/ui` |
```

Comment:
Nice change! Sadly this link doesn't work yet :(

Reply:
It will once netlify builds work again ;)